src/components/ImageView/index.js

```diff
@@ -243,15 +237,17 @@ class ImageView extends PureComponent {
         if (this.canUseTouchScreen) {
             return (
                 <View
+                    ref={el => this.scrollableRef = el}
                     style={[styles.imageViewContainer, styles.overflowHidden]}
+                    onLayout={this.onContainerLayoutChanged}
                 >
                     <Image
                         source={{uri: this.props.url}}
                         style={[
                             styles.w100,
                             styles.h100,
                         ]}
-                        resizeMode="contain"
+                        resizeMode={this.state.zoomScale >= 1 ? 'center' : 'contain'}
```

Comment:
Why do we need to dynamically check and set the resizeMode here?  Isn't `center` will work? 

Can you show me videos to indicate the difference it makes?

Reply:
You are absolutely right 😮. I focused on fixing the version that works with the previous big image and missed it. This will change a lot of things thank you so much 🙏 . We dont need a lot of changes no more 🙏 🙏 . 

Note : To get the zoomScale information, I made changes in `componentDidMount` and added the View `onLayout`and `ref` callbacks.